code/modules/medical/cloning.dm

```diff
@@ -30,6 +30,9 @@
 
 	l_color = "#7BF9FF"
 
+/obj/machinery/cloning/clonepod/empty
```

Comment:
Dude I'm not turning this PR into a map PR, how many of these do we have open already?